fixes.inc

```diff
@@ -7707,6 +7812,27 @@ native BAD_TogglePlayerControllable(playerid, toggle) = TogglePlayerControllable
 		//  END: PassengerSeating
 		// =============================
 
+		// =============================
+		//  BEGIN: SilentTeleport
+		// =============================
+		#if FIX_SilentTeleport
```

Comment:
Same, you forgot to add check for definition of `FIX_SilentTeleport` before `OnPlayerEnterVehicle`.

Reply:
Done.